openstack_controller/tests/fixtures/POST/identity/v3/auth/tokens/1e6e233e637d4d55a50a62b63398ad15.json

```diff
@@ -187,7 +187,7 @@
             "id": "168f02a638de42f6b66b60b01f905830",
             "interface": "public",
             "region_id": "RegionOne",
-            "url": "http://127.0.0.1:8004/heat-api",
+            "url": "http://127.0.0.1:8004/heat-api/v1/1e6e233e637d4d55a50a62b63398ad15",
```

Comment:
Do we have a test where the endpoint does not have the project ID in it?

Reply:
We did not, so I added `http://127.0.0.1:8004/heat-api` to the list of system-level endpoints, and made sure that the service check works properly for this endpoint (this increased the number of api calls made by `test_response_time_exception` as a side effect).

Reply:
makes sense, thanks!